package.json

```diff
@@ -1,11 +1,12 @@
 {
-  "name": "resumeapp",
+  "name": "ResumeBuilder",
```

Comment:
This is where you add the api you're using right?

Reply:
Yes, all installed packages go into package.json

The reason I changed the name here was because when I used create react app to get all the boilerplate files, I named it resumeapp and wanted to change it